ethcore/src/evm/evm.rs

```diff
@@ -79,6 +82,12 @@ impl From<builtin::Error> for Error {
 	}	
 }
 
+impl From<wasm::RuntimeError> for Error {
+	fn from(_err: wasm::RuntimeError) -> Self {
+		Error::Wasm("Runtime error")
```

Comment:
This isn't a really useful message. Why not format the wasm error into a `String`?